lib/linguist/vendor.yml

```diff
@@ -60,10 +60,17 @@
 # Bootstrap css and js
 - (^|/)bootstrap([^.]*)\.(js|css|less|scss|styl)$
 - (^|/)custom\.bootstrap([^\s]*)(js|css|less|scss|styl)$
+# Bootstrap with a directory between the files and the main folder like bootstrap/css/bootstrap.css
+- (^|/)bootstrap/(css|js)/.*\.(js|css|less|scss|styl)$
 
 # Font Awesome
 - (^|/)font-awesome\.(css|less|scss|styl)$
 - (^|/)font-awesome/.*\.(css|less|scss|styl)$
+- (^|/)fontawesome\.(css|less|scss|styl)$
+- (^|/)fontawesome/.*\.(css|less|scss|styl)$
+# Font Awesome with a directory between the files and the main folder like fontawesome/css/fontawesome.all.css
+- (^|/)font-awesome/.*/.*\.(js|css|less|scss|styl)$
+- (^|/)fontawesome/.*/.*\.(js|css|less|scss|styl)$
```

Comment:
This shouldn't be needed as your additions above will catch these situations too.